src/common/components/discussion/index.tsx

```diff
@@ -66,23 +66,38 @@ import { ProfilePopover } from "../profile-popover";
 interface ItemBodyProps {
   entry: Entry;
   global: Global;
+  isRawContent: boolean;
 }
 
 export class ItemBody extends Component<ItemBodyProps> {
   shouldComponentUpdate(nextProps: Readonly<ItemBodyProps>): boolean {
     return this.props.entry.body !== nextProps.entry.body;
   }
 
+  componentDidUpdate(
+    prevProps: Readonly<ItemBodyProps>,
+    prevState: Readonly<{}>,
+    snapshot?: any
+  ): void {
+    // console.log("Componentdid update");
+    // console.log(this.props);
+  }
```

Comment:
Why it needs?

Reply:
@dkildar  As discussed with Feruz this issue is on hold and we are still developing. I have assigned it back to me and will assign the PR to you and Feruz when its ready.